.github/workflows/vale.yml

```diff
@@ -3,8 +3,7 @@ name: Vale
 on:
   pull_request:
     paths:
-      - 'guides/common/*.adoc'
-      - '.github/workflows/vale.yml'
+      - '**.adoc'
```

Comment:
Please separate these changes, at least to its own commit.

Reply:
Done.